embedchain/chunkers/text.py

```diff
@@ -14,3 +14,18 @@ class TextChunker(BaseChunker):
     def __init__(self):
         text_splitter = RecursiveCharacterTextSplitter(**TEXT_SPLITTER_CHUNK_PARAMS)
         super().__init__(text_splitter)
+
+    def create_chunks(self, loader, url):
+        if os.path.isfile(url):
+            with open(url, 'r') as file:
+                content = file.read()
+        else:
+            content = loader.load_data(url)[0]["content"]
```

Comment:
`loader.load_data(url)` should already return the full content. No need to reach further down to the return object